src/Content/AttachmentLocal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Phlib\Mail\Content;
+
+use Phlib\Mail\Exception\RuntimeException;
+
+/**
+ * Attachment class used to add local files to a Mail object
+ *
+ * @package Phlib\Mail
+ */
+class AttachmentLocal extends Attachment
```

Comment:
Sounds more like a static create method than a separate class to me. You've almost said it yourself in that comment, `AttachmentFromFile` and "create a new part".